src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
At the moment, we don't call register() on the service worker in index.js. So we could remove this file if we don't intent to use a service worker

Reply:
On it (like a car bonnet)